docs/IE_PLUGIN_DG/NewTransformation.md

```diff
@@ -0,0 +1,15 @@
+# Writing ngraph transformations {#new_ngraph_transformation}
```

Comment:
nit: I would suggest to use just ngraph_transformations as a tag. It's not critical and I can change it later.

Reply:
> Do we build this documentation in the CI?

No, there is no CI for public docs yet.